.env.template

```diff
@@ -18,3 +18,9 @@ ADDRESS_TO_REMOVE=...
 
 # RPC URLs for different networks
 RPC_URL=...
+
+## API key to verify contracts
+EXPLORER_API_KEY=
+
+# Verification API V2 (true/false)
+IS_VERIFICATION_API_V2=...
```

Comment:
I don't think we need that 

<img width="991" height="241" alt="image" src="https://github.com/user-attachments/assets/03ed2b82-03a1-43db-b0f5-09a0d4873bb7" />